pkg/storaged/lvol-tabs.jsx

```diff
@@ -145,7 +160,7 @@ function lvol_and_fsys_resize(client, lvol, size, offline, passphrase) {
 
     function lvm_resize() {
         if (size != lvol.Size)
-            return lvol.Resize(size, { });
+            return lvol.Resize(size, { pvs: pvs ? { t: 'ao', v: pvs } : undefined });
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20230912-152256-2d3c74b2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)